src/Sylius/Bundle/CoreBundle/CatalogPromotion/Validator/CatalogPromotionAction/FixedDiscountActionValidator.php

```diff
@@ -54,6 +54,10 @@ public function validate(array $configuration, Constraint $constraint, Execution
         }
 
         foreach ($configuration as $channelCode => $channelConfiguration) {
+            if (null === $channelConfiguration['amount']) {
+                $context->buildViolation('sylius.catalog_promotion_action.fixed_discount.channel_not_configured')->atPath('configuration')->addViolation();
+            }
```

Comment:
`isChannelConfigured` method, that is used above, checks the amount for enabled channels so this `if` shouldn't be needed I guess